### DISCUSSION TITLE
Add ability to skin the time-signature based note colors

### DISCUSSION THREAD

**Discussion by cesarila** [ppy/osu#15297] [Ideas] [answered]
Currently, it looks like the game has some default colors based on the beat snap divisor defined here:

https://github.com/ppy/osu/blob/01a86f4c3b6e8b5dd687c0ecb6cce1c5a5ea33d1/osu.Game/Screens/Edit/BindableBeatDivisor.cs#L52-L83

And those colors are used with the the timing-based note coloring, seen here:

https://github.com/ppy/osu/blob/01a86f4c3b6e8b5dd687c0ecb6cce1c5a5ea33d1/osu.Game.Rulesets.Mania/Objects/Drawables/DrawableNote.cs#L122

It would be really nice to be able to use colors of my choosing with that feature.

**Comment by bdach** [comment, marked as answer]
probably still a while until that will be implemented, if it even will be. prerequisite is having a new skinning system in place, i don't think the existing one is going to get extended for that.

**Reply by cesarila** [reply]
Fair enough. I suppose that I can build from source with different colors if I really care that much.
